codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/BuilderGenerator.kt

```diff
@@ -50,14 +51,15 @@ class OperationBuildError(private val runtimeConfig: RuntimeConfig) {
 /** setter names will never hit a reserved word and therefore never need escaping */
 fun MemberShape.setterName(): String = "set_${this.memberName.toSnakeCase()}"
 
-class BuilderGenerator(
-    val model: Model,
-    private val symbolProvider: RustSymbolProvider,
-    private val shape: StructureShape
+open class BuilderGenerator(
+    protected val model: Model,
+    protected val symbolProvider: RustSymbolProvider,
+    protected val shape: StructureShape
 ) {
```

Comment:
I should be able to revert this change in this PR, but it may be needed later for changing input stream types to `Into<EventInputStream>`, unless there's a way to do that (only for builder functions) from the symbol provider.